src/index.ts

```diff
@@ -213,6 +213,40 @@ server.tool(
   },
 );
 
+server.tool(
+  "push_api_limit",
+  "Push message API is limited messages per month. This tool provides the current limit and remaining quota.",
+  {},
+  async () => {
+    const response = await messagingApiClient.getMessageQuota();
+    return {
+      content: [
+        {
+          type: "text",
+          text: JSON.stringify(response),
+        },
+      ],
+    };
+  },
+);
+
+server.tool(
+  "push_api_total_usage",
```

Comment:
It's not limited to push; it includes push + multicast + narrowcast + broadcast. How about considering it simply as a message quota?